frontend/device/pocketbook/device.lua

```diff
@@ -58,26 +20,23 @@ local function no() return false end
 local PocketBook = Generic:new{
     model = "PocketBook",
     isPocketBook = yes,
-    isInBackGround = false,
     hasOTAUpdates = yes,
     hasWifiToggle = yes,
     isTouchDevice = yes,
     hasKeys = yes,
     hasFrontlight = yes,
     canSuspend = no,
-    emu_events_dev = "/dev/shm/emu_events",
```

Comment:
@ezdiy: I see a point in keeping that fifo. It turns out that the Pocketbook uses a new instance each time a book is opened from the PB Library.  https://github.com/koreader/koreader/issues/6321. We can repurpose the same instance feeding the new file as ev.value with our own ev.code, for example:

Instead of having the same shell script duplicated in `/mnt/ext1/applications/koreader.app` and `/mnt/ext1/system/bin/koreader.app`, replace the later with

```bash
#!/bin/sh

# IPC for pocketbook. Used to redirect commands to a running instance.
# Events generated with this script need to be handled by handleMiscEv.

# $1: full path of the file to open

KO_EVENT_PIPE=/dev/shm/emu_events
KO_SCRIPT=/mnt/ext1/applications/koreader.app
KO_DIR=/mnt/ext1/applications/koreader

# we're always starting from our working directory
cd "${KO_DIR}" || exit

NewMiscEv() {
    ./luajit <<EOF
    require("setupkoenv")
    local dump = require("dump")
    local ffi = require("ffi")
    local ev_time = ffi.new("timeval")
    ffi.C.gettimeofday(ev_time, nil)
    local ev = {
        type = 4,
        code = 999,
        value = "${1}",
        time = { sec = tonumber(ev_time.tv_sec), usec = tonumber(ev_time.tv_usec) }
    }
    local pipe = io.open(KO_EVENT_PIPE, "wb")
    if pipe then
        pipe:write(dump(ev))
        pipe:close()
    end
EOF
}


if [ -p "${KO_EVENT_PIPE}" ]; then
    NewMiscEv "${@}"
else
    "${KO_SCRIPT}" "${@}"
fi
```

Probably there are better ways of doing the same and I'm fine with all of them :D, including leave this bug unfixed, but this is just FYI.

Reply:
@pazos Noted. Definitely wontfix in this PR as there's no sensible way to revive the pipe just to inject events. Scope of future ones though - https://github.com/koreader/koreader/issues/6597